scripts/install.sh

```diff
@@ -391,10 +391,17 @@ sed_in_place "/.*# avalanche completion/d" $BASH_COMPLETION_MAIN
 echo "source $BASH_COMPLETION_SCRIPT_PATH # avalanche completion" >> $BASH_COMPLETION_MAIN
 if [ $(uname) = Darwin ]
 then
-    BASHRC=~/.bashrc
-    touch $BASHRC
-    sed_in_place "/.*# avalanche completion/d" $BASHRC
-    echo "source $(brew --prefix)/etc/bash_completion # avalanche completion" >> $BASHRC
+    BREW_INSTALLED=true
+    which brew > /dev/null 2> /dev/null || BREW_INSTALLED=false
```

Comment:
Why do we need this double indirection? Can we do this? `which brew &>/dev/null` ?

Reply:
we are in general using `sh` not `bash` , `sh` in general does not support that redirection

Reply:
you are right